mpt_extension_sdk/__init__.py

```
@@ -1,3 +1,10 @@
-from mpt_extension_sdk.extension_app import ExtensionApp, ExtensionRouter
+from mpt_extension_sdk.extension_app import ExtensionApp
+from mpt_extension_sdk.routing import ApiRouter, EventRouter, PlugRouter, ScheduleRouter
 
-__all__ = ["ExtensionApp", "ExtensionRouter"]  # noqa: WPS410
+__all__ = [  # noqa: WPS410
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS410

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)